specification/sigma-rules-specification.md

```diff
@@ -207,13 +209,16 @@ The goal is to improve the readability of correlation rules.
 
 Defines the taxonomy used in the Sigma rule. A taxonomy can define:
 
-* field names, example: `process_command_line` instead of `CommandLine`.
-* field values, example: a field `image_file_name` that only contains a file name like `example.exe` and is transformed into `ImageFile: *\\example.exe`.
-* logsource names, example: `category: ProcessCreation` instead of `category: process_creation`
+- field names, example: `process_command_line` instead of `CommandLine`.
+- field values, example: a field `image_file_name` that only contains a file name like `example.exe` and is transformed into `ImageFile: *\\example.exe`.
+- logsource names, example: `category: ProcessCreation` instead of `category: process_creation`
 
 The Default taxonomy is `sigma`. A custom taxonomy must be handled by the used tool or transformed into the default taxonomy.
 
-More information on the default taxonomy can be found in the [Sigma Taxonomy Appendix](/appendix/sigma-taxonomy-appendix.md) file.
+More information on the default taxonomy can be found in the [Sigma Taxonomy Appendix](sigma-appendix-taxonomy.md) file.
+Custom taxonomy can be found in the [Sigma Custom Taxonomy Appendix](sigma-appendix-custom-taxonomy.md) file.
+
+For example in pySigma for ocsf taxonomy you can use the [pySigma-pipeline-ocsf pipeline](https://github.com/SigmaHQ/pySigma-pipeline-ocsf)
```

Comment:
Perhaps we should use the custom taxonomy appendix to refer such taxonomy projects.